inc/saiacl.h

```diff
@@ -951,6 +951,42 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_FIELD_MACSEC_SCI,
 
+    /**
+     * @brief A list of indexes into MPLS label stack (0 is top-of-the-stack)
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default false
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL_INDEX_LIST,
```

Comment:
when we create acl table, we add this match field, at that point of time, the SAI needs to know the width of the fields so that it can allocate the enough tcam resource for this table. the question here is it is unclear the width of field from this attribute.

Reply:
One suggestion is to add a separate attribute for each MPLS label instead of a list. Similar to what is being proposed for Hash attributes. That will allow SAI driver to unambiguously determine the  tcam resources and also if underlying hardware has the ability to match on that particular label.

Reply:
thanks, will apply these comments

- SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL0
- SAI_ACL_TABLE_ATTR_FIELD_MPLS_LABEL0

Reply:
Yes, I didn't take into account that the current definition is not deterministic in terms of the key size. Expanding the array as suggested will solve the problem.

Reply:
@ashutosh-agrawal @lguohan @marian-pritsak applied comments